plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -575,13 +575,12 @@ class read_only {
       };
    }
 
-   struct push_ro_transaction_params {
-      name account_name;
-      bool return_failure_traces = true;
+   struct send_ro_transaction_params_v1 {
+      bool return_failure_traces = false;
```

Comment:
We want the default to be `true` for develop-boxed.

Reply:
made it true by default via cleos

Reply:
Would also like it true by default here for EOSJS and curl, etc.